func_tests/test_client.py

```diff
@@ -73,3 +75,70 @@ def test_batch_change(vinyldns_test_context):
     assert change2.record.ptrdname == 'change-test.vinyldns.'
     assert change2.type == RecordType.PTR
 
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[0].zone_id, r.changes[0].record_set_id)
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[1].zone_id, r.changes[1].record_set_id)
+
+
+def test_batch_change_review_process_reject(vinyldns_test_context):
+    changes = [
+        AddRecord('test-approve-success.not.loaded.', RecordType.A, 200, AData("4.3.2.1"))
```

Comment:
not loaded zone has to be pulled into `docker/bind9`, be sure to update the named.conf as well

Reply:
doh, never mind